src/Appwrite/Messaging/Adapter/Realtime.php

```diff
@@ -149,7 +149,7 @@ public static function send(string $projectId, array $payload, array $events, ar
             'data' => [
                 'events' => $events,
                 'channels' => $channels,
-                'timestamp' => DateTime::now(),
+                'timestamp' => date('c'),
```

Comment:
I know it's longer than what you have, but let's do the following to be consistent with what we do in the rest of the codebase:

```suggestion
                'timestamp' => DateTime::formatTz(DateTime::now()),
```